frame/multisig/src/lib.rs

```diff
@@ -257,7 +257,7 @@ pub mod pallet {
 			let dispatch_info = call.get_dispatch_info();
 			(
 				T::WeightInfo::as_multi_threshold_1(call.using_encoded(|c| c.len() as u32))
-					.saturating_add(dispatch_info.weight)
+					.saturating_add(dispatch_info.weight.computation)
```

Comment:
I think we should `impl ops::Add for WeightV2` rather than ignoring the `bandwidth` here.

Reply:
that is not the problem, the problem is that this weightinfo formula is giving us a u64